evidence/characters/electro/fischl.md

```diff
@@ -398,3 +398,16 @@ Recommended Artifact Mainstats:
 **Significance:**  
 Better weapon selection for support Fischl.
 
+## Fischl Ult Has I-Frames
+**By:** !K#3689 
+
+**Added:** 5/6/2021
+
+[Discussion](https://tickettool.xyz/direct?url=https://cdn.discordapp.com/attachments/839862552401674240/839936289129365625/transcript-fischl-ult-can-iframe.html)
+
+**Finding:** Fischl ult has a tiny window that you can use to iframe through attacks.
+
+**Evidence:** https://youtu.be/7RPfSWwUOLM 
```

Comment:
URL not hyperlinked. Consider the following:
`[Video showing no HP loss when ruin guard lands](https://youtu.be/7RPfSWwUOLM0)`